packages/react/src/hooks/useCallbackOnce/useCallbackOnce.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect, vi } from 'vitest';
+import { renderHook, act } from '@testing-library/react';
+import { useCallbackOnce } from '.';
+
+describe('useCallbackOnce', () => {
+  it('콜백이 한 번만 실행되어야 합니다.', () => {
+    const callback = vi.fn();
+    const { result } = renderHook(() => useCallbackOnce(callback));
+
+    act(() => {
+      result.current();
+      result.current();
+      result.current();
+    });
```

Comment:
```suggestion
    await waitFor(() => {
      result.current();
      result.current();
      result.current();
    });
```
act대신 @testing-library/react의 waitFor를 사용해주세요 ~!